src/components/molecules/LoginForm.tsx

```diff
@@ -9,23 +9,23 @@ import AppFormInput from "../atom/AppFormInput";
 const LoginForm = () => {
     const {user, onChange, hasError, onSubmit} = useLoginForm(
         {
-            id: "",
+            email: "",
             password: ""
         });
     const {showPassword, toggleShowPassword} = useShowPassword();
     const {t: translate} = useTranslation(namespaces.pages.loginScreen);
 
-    const buttonDisabled = user.id.length !== 9 || user.password.length < 3 || hasError;
+    const buttonDisabled = user.email.length < 6  || user.password.length < 3 || hasError;
```

Comment:
I would try to avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,see%20List%20of%20file%20signatures), they get more and more dangerous as the time pass.

Furthermore, it could be useful to create an input validator for this. In any case, that `6` could be something like:
```
const EMAIL_MINIMUM_LENGTH = 6;
const PASSWORD_MINIMUM_LENGTH = 3;
```

Just out of curiosity, why 6 for emails?

Reply:
I agree with you, I don't do magic numbers. I just took the validation from another email component because I needed to test this integrated with firebase. 

It wasn't important for me the email or password I just wanted to test this integrated with firebase, because this login is going to change to phone number, so email or password are not important anymore, just the logic about connecting this with firebase

I don't understand the 6 neither, I took it from another email component because as I said I am going to change this login to phone number in my next PR so it wasn't important for me at that time

Is that ok for you, or do you want me to fix the email validation anyways?

Reply:
it is fine if it is being used in other parts of the project, but sometimes it is worth investing some time into ensuring that we understand what those numbers are. I think for this PR sounds ok though

Reply:
Yes, the Boy Scout law, I should have checked those numbers, but we are planning to replace this singIn with a phone signIn then I didn't want to spend more time in the password/email when we are going to remove it. Thank you for your comments! I agree with you